### ISSUE TITLE
PYMC 2 warning

### ISSUE THREAD

**Issue by frictionPG** [pymc-devs/pymc2#196]
This isn't really an issue, but if you google PYMC the first result is the instalation instructions for PYMC2. It seems like it would be worth adding some sort of warning to the documentation of PYMC2 to say that PYMC3 is out and should be installed unless you have a good reason not to.

**Comment by twiecki** [comment]
That is a great point.

**Comment by twiecki** [comment]
@frictionPG want to make a PR to update the docs by any chance?

**Comment by fonnesbeck** [comment]
There is a notice at the very top of the repo, as well as a note at the top of the README, so I had hoped that would be sufficient. If we really want to be hardcore about it, we could display a message when the package is imported.

**Comment by mikeWShef** [comment]
Sorry I should have started this from my personal account (this one). I don't think it is really being 'hard core' it's just putting the message where it will be seen. I spent a long time following the instructions, reinastalling compilers etc. only to end up seeing a warning about installing it at all on the issues pages here (which makes it seem like I am not the only one).

**Comment by twiecki** [comment]
@mikeWShef so you wanted to get started on pymc3, typed pymc into the google search, went to the pymc2 site and followed the installation instructions?

**Comment by mikeWShef** [comment]
Yes, basically. I had read about it previously, so i googled 'pymc' to get the current version but the first hit on google is the pymc2 documentation and installiation instructions. I wasn't aware that pymc3 was a separate thing, I am a totally new user.

**Comment by twiecki** [comment]
Thanks, that's very helpful. Ideally we could get google to show pymc3 as the first search result but that probably can't be done.

**Comment by fonnesbeck** [comment]
Another strategy might be to get rid of the online docs for PyMC2 altogether and just provide a PDF version in the repo for those that need it. This would halt its appearance first when searching for PyMC, and reduce accidental discovery of the wrong docs.
